bool exist_rob_flag
bool exist_obs_flag
int32 rob_num
int32 obs_num
float64[5] rob_cam_pos_x
float64[5] rob_cam_pos_y
float64[5] rob_cam_vel_x
float64[5] rob_cam_vel_y
float64[4] obs_cam_pos_x
float64[4] obs_cam_pos_y
float64[4] obs_cam_vel_x
float64[4] obs_cam_vel_y